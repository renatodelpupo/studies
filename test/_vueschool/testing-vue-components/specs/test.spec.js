import { mount } from '@vue/test-utils'
import TestComponent from '@/test.vue'

test('mount a vue component', () => {
  const wrapper = mount(TestComponent, {
    propsData: {
      value: 'VueSchool',
    },
  })

  expect(wrapper.html()).toMatchSnapshot()
})
