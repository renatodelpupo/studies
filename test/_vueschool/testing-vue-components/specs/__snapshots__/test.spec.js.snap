// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount a vue component 1`] = `
<div>
  <h1>Hello VueSchool</h1>
</div>
`;
